app/src/main/java/edu/metrostate/cardealer/activity/DealerListActivity.java

```diff
@@ -0,0 +1,60 @@
+package edu.metrostate.cardealer.activity;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+//import android.app.AlertDialog;
+//import android.app.Dialog;
+import android.content.Intent;
+import android.os.Bundle;
+//import android.view.View;
+//import android.widget.AdapterView;
+import android.widget.ListView;
+
+import edu.metrostate.cardealer.CarDealerApplication;
+import edu.metrostate.cardealer.R;
+import edu.metrostate.cardealer.adapter.DealerAdapter;
+
+public class DealerListActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dealer_list);
+
+        // Get the application instance from the activity
+        CarDealerApplication app = (CarDealerApplication) getApplication();
+
+        // Create an adapter for the list view
+        DealerAdapter adapter = new DealerAdapter(this, app.getDealerList());
```

Comment:
Planning on changing app.getDealerList() to StateManager.dealerGroup.getDealers()